### PR TITLE
Tag db improvements

### PR DESCRIPTION
**Pull request by Marie000**

### CODE REVIEW

File: migrations/20190204095522_tag.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 exports.up = function (knex, Promise) {
   return knex.schema.createTable('Tag', function (table) {
-    table.uuid('id').primary()
+    table.increments()
```
Comment (Marie000) [inline migrations/20190204095522_tag.js]: any objection to simple incremental ids?

File: routes/outbox-post.js [hunk 2]
```diff
@@ -149,6 +149,23 @@ module.exports = function (app) {
             pr
               .then(result => {
                 debug(result)
+                // catching duplicate entries to publication_tag table
```
Comment (Marie000) [inline routes/outbox-post.js]: I'm not crazy about having this here, but I don't see a better solution.

**Comment by Marie000** [comment]
also addresses issue #153

**Comment by Marie000** [comment]
Phew. Finally all works.
I just spent a few hours struggling to understand why my integration tests were working locally but failing on Travis (both using postgresql)
Then I realized it was the unit tests failing on Travis. Not the integration tests. 

So I am officially an idiot.

**Comment by baldurbjarnason** [comment]
Been there. (Done many, many things easily equally embarrassing.) 🙂 

Looks good. 👍

**Pull request merged**